include/rocksdb/statistics.h

```diff
@@ -411,6 +411,9 @@ struct HistogramData {
   double percentile99;
   double average;
   double standard_deviation;
+  // zero-initialize new members since old Statistics::histogramData()
```

Comment:
LGTM, but I don't understand this comment. What is the problem with old implementations ?